editor/components/block-list/block.js

```diff
@@ -452,6 +465,7 @@ export class BlockListBlock extends Component {
 			'is-multi-selected': isMultiSelected,
 			'is-hovered': isHovered,
 			'is-reusable': isReusableBlock( blockType ),
+			'is-hidden': dragging,
```

Comment:
Is this a concern of the consumer of `Draggable` ? Seems to require a lot of work to use.

Reply:
That is correct - showing an overlay in place of the original content or some other style deviation is the concern of the consumer.